test/e2e/network/networking.go

```diff
@@ -477,7 +477,7 @@ var _ = SIGDescribe("Networking", func() {
 		// restart iptables"?). So instead we just manually delete all "KUBE-"
 		// chains.
 
-		ginkgo.By("dumping iptables rules on a node")
+		ginkgo.By("dumping iptables rules on node " + host)
```

Comment:
I would also log the result, so we can see what iptables rules are there
```
framework.Logf("host %s iptables rules, %s",  host, result)
```

Reply:
But there are normally _a lot_ of iptables rules, and they're normally interesting for debugging.

Though I guess maybe since this is a `[Disruptive]` test and so only runs in periodics maybe we should err on the side of more pre-emptive debugging...

Reply:
I didn't notice that you are already logging the `iptables-save -t nat` below, that should be enough